.cmake-format.py

```diff
@@ -0,0 +1,221 @@
+# ----------------------------------
+# Options affecting listfile parsing
+# ----------------------------------
+with section("parse"):
+    # Specify structure for custom cmake functions
+    additional_commands = {'foo': {'flags': ['BAR', 'BAZ'],
```

Comment:
lol?

Reply:
I literally did `dump-config` and this is what it output.

Reply:
Fixed.